portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,65 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Toni!</h1>");
+    //Initializing and adding in arrayList
+
+    private ArrayList<String> commentsArr;
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    commentsArr = new ArrayList<String>();
+    commentsArr.add(request.getParameter("fname"));
+    commentsArr.add(request.getParameter("lname"));
+    commentsArr.add(request.getParameter("comment"));
+
+    
+    response.getWriter().println("Thank you for your feedback!");
+        response.setContentType("text/html;");
+
+    //Creating an entity
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comments", commentsArr);
+
+    //Storing the Entity in datastore
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+}
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
is this empty function needed?

Reply:
When I try to delete it, I get errors. So that's I was keeping it there because it wasn't causing any errors.

Reply:
Hey Toni, What error are you getting when you remove this method? (I couldn't find my previous comment and reply)